pkgs/development/haskell-modules/configuration-ghc-8.0.x.nix

```diff
@@ -2,6 +2,9 @@
 
 with import ./lib.nix { inherit pkgs; };
 
+let haskell = pkgs.haskell;
+in
```

Comment:
You seem to use that namespace for accessing `haskell.lib` only, but `haskell.lib` is in scope anyway because of line 3. Am I missing something?